src/modules/endpoint/DatabaseConnector.js

```diff
@@ -75,6 +88,39 @@ class DatabaseConnector {
     return getResourceCacheFilesPath(context.cityCode)
   }
 
+  getLastUpdatePath (): string {
+    return `${CACHE_DIR_PATH}/cities.json`
+  }
+
+  async storeLastUpdate (lastUpdate: Moment, context: DatabaseContext) {
+    const path = this.getLastUpdatePath()
+
+    let currentCityMetaData: MetaCitiesJsonType = {}
+    const fileExists: boolean = await RNFetchblob.fs.exists(path)
+    if (fileExists) {
+      currentCityMetaData = JSON.parse(await this.readFile(path))
+    }
+
+    const lastUpdatePath = `${context.cityCode}.languages.${context.languageCode}.lastUpdate`
+    set(currentCityMetaData, lastUpdatePath, lastUpdate)
```

Comment:
Why are you using set() here instead of using the "normal" syntax

Reply:
If a portion of the path does not exist, `set` creates it. Otherwise I have to create it by hand first and then set the `lastUpdate`.